src/App.tsx

```diff
@@ -180,8 +188,23 @@ const App = () => {
                   handleSubmit={handleProposal("coreEvalProposal")}
                   titleDescOnly={false}
                   title="/agoric.swingset.CoreEvalProposal"
-                  description="This is a governance proposal that executes code. You will need to provide a JS Bundle, and a JSON Permit file."
                   msgType="coreEvalProposal"
+                  description={
+                    <>
+                      This is a governance proposal that executes code after a
+                      passing vote. The JSON Permit grants{" "}
+                      <Link href="https://docs.agoric.com/guides/coreeval/permissions.html">
+                        capabilities
+                      </Link>{" "}
```

Comment:
We seem to be able to use `<p>` and `<div>`. Why not `<a>`? Why is a special `<Link>` tag needed?
OK, I see `target="_blank" rel="noopener noreferrer"`. We can't let people make up their own minds about which tab to open links in? We're trying to hide where the links come from? (hosting on https will do that for us, no?)

Reply:
This is a good point, I guess it's only appropriate when paired with an external link icon. `noreferrer` is old relic, that used to be security a requirement when target="_blank".

Updating these to <a href="..." />